tests/unit/v1/test_async_client.py

```diff
@@ -200,7 +201,8 @@ async def test_collections(self):
 
         collection_ids = ["users", "projects"]
         client = self._make_default_one()
-        firestore_api = mock.Mock(spec=["list_collection_ids"])
+        firestore_api = asyncmock.AsyncMock()
+        firestore_api.mock_add_spec(spec=["list_collection_ids"])
```

Comment:
If someone could take a look at this test case it would be appreciated. For some reason the final assertion in the test case is failing, stating that the mock is called 0 times, even though it is fairly evident the mock is being used from the fact that the previous assertions pass.

Reply:
Turns out there's some issue with `asyncmock` and using `assert_called_once_with`.